modules/monitoring-support-considerations.adoc

```diff
@@ -18,7 +18,7 @@ The Alertmanager configuration is deployed as a secret resource in the `openshif
 * *Modifying resources of the stack.* The {product-title} monitoring stack ensures its resources are always in the state it expects them to be. If they are modified, the stack will reset them.
 * *Deploying user-defined workloads to `openshift-&#42;`, and `kube-&#42;` projects.* These projects are reserved for Red Hat provided components and they should not be used for user-defined workloads.
 * *Modifying the monitoring stack Grafana instance.*
-* *Installing custom Prometheus instances on {product-title}.*
+* *Installing custom Prometheus instances on {product-title}.* A custom instance is a Prometheus custom resource (CR) managed by the Prometheus Operator.
```

Comment:
managed by the Prometheus Operator
->
managed by the OLM Prometheus Operator

Reply:
"managed by the Prometheus Operator" is fine by me. We don't want users to deploy additional Prometheus objects in the openshift-monitoring or openshift-user-workload-monitoring namespaces either.

Reply:
@juzhao Do you agree with @simonpasquier? If so, please give this PR an lgtm. If not, why not?

Reply:
thanks, LGTM